react-app/ccf/src/pages/admin-database/AdminDatabase.css

```
@@ -310,133 +196,159 @@
+    text-transform: uppercase;
+    letter-spacing: 0.04em;
+    margin: 0;
+}
+
+.admin-database-page .admin-detail-value {
+    color: var(--ccf-text);
+    font-size: 15px;
+    font-weight: 500;
+    line-height: 1.4;
+    margin: 0;
+    word-break: break-word;
```

Stylelint (17.11.1)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)